src/shared/test/cfi-test.js

```diff
@@ -41,7 +41,9 @@ describe('sidebar/util/cfi', () => {
         expected: ['/2/4', '/2/4'],
       },
     ].forEach(({ range, expected }) => {
-      assert.deepEqual(splitCFIRange(range), expected);
+      it('splitCFIRange', () => {
+        assert.deepEqual(splitCFIRange(range), expected);
+      });
```

Comment:
Vitest complained about an empty `describe`, and then I realized the actual `it` block was accidentally overlooked here.

I'm not sure what would be the right description though. We should change it before merging this.

Reply:
"returns expected splits" or "splits range into expected pieces" would be fine.